src/js/views.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+var $ = window.$;
+var L = window.L;
+var _ = window._;
+var Backbone = window.Backbone;
+
+window.MapView = Backbone.View.extend({
+    initialize: function(options) {
+        var map = L.map('map', {
+            center: options.model.get('initialMapCenter'),
+            zoom: options.model.get('initialMapZoom'),
+        });
+
+        L.tileLayer(options.model.get('mapTilesUrl'), {
+            attribution: options.model.get('mapTilesAttribution'),
+            minZoom: options.model.get('mapTilesMinZoom'),
+            maxZoom: options.model.get('mapTilesMaxZoom'),
+            ext: 'png',
+        }).addTo(map);
+
+        this.map = map;
+        this.model = options.model;
+        this.listenTo(this.model, 'change:selectedRegion', this.render);
+    },
+
+    render: function() {
+        var newCenter = this.model.get('selectedRegion') ?
+            this.model.get('selectedRegion').get('mapCenter') :
+            this.model.get('initialMapCenter');
+
+        var newZoom = this.model.get('selectedRegion') ?
+            this.model.get('selectedRegion').get('mapZoom') :
+            this.model.get('initialMapZoom');
+
+        this.map.flyTo(newCenter, newZoom);
+
+        return this;
+    },
+});
+
+window.RegionListView = Backbone.View.extend({
+    el: '#region-list-view',
+
+    template: _.template($('#regionListTemplate').html()),
+
+    events: {
+        'click': 'handleSelection',
+    },
+
+    initialize: function(options) {
+        this.model = options.model;
```

Comment:
This should happen automatically if it's assigned on initialization, which you are doing above:

```
RegionListView({ model: this.model });
```

Reply:
From the backbone documentation:

```
There are several special options that, if passed, will be attached directly to the view: model, collection, el, id, className, tagName, attributes and events.
```